content/en/getting_started/integrations.md

```diff
@@ -0,0 +1,138 @@
+
+### Installation
+
+If you want to connect with a cloud service provider, navigate to that provider on the [Integrations page][16] for specific instructions on how to connect. For most other integrations, start by installing the [Datadog Agent][17]. We also have containerized agents: [docker][18], and [kuberneties][19]. After you've downloaded the Agent, go to the [Integrations page][16] section to find specific configuration instructions for individual integrations.
+
+### Configuration
+
+Configurations are more specifc to [individual integrations][16]. Configure most integrations by editing the `conf.yaml` file, in the `conf.d/` folder at the root of your Agent's configuration directory. Most integration have a sample `conf.yaml` that lists all available configuration options.
+
+For example, this is a sample `conf.yaml` for the `http_check` integration:
+
+```
+init_config:
+
+instances:
+  - name: Example website
+    url: https://example.com/
+    # disable_ssl_validation: false      # default is true, so set false to check SSL validation
+    # ca_certs: /path/to/ca/file         # e.g. /etc/ssl/certs/ca-certificates.crt
+    # check_certificate_expiration: true # default is true
+    # days_warning: 28                   # default 14
+    # days_critical: 14                  # default 7
+    # timeout: 3                         # in seconds. Default is 10.
+  - name: Example website (staging)
+    url: http://staging.example.com/
+```
+
+### Validation
+
+To validate your configuration, [run the Agent's `status` subcommand][20], and look for new configuration under the Checks section.
+
+### Tagging
+
+Tagging is a key part of filtering and aggregating the data coming into Datadog across many sources. You can assign tags in configuration files, in environment variables, in the UI, or the API, in DogStatsD, and new integrations automatically inherited existing tags. For more information about tagging, see [Getting started with tags][12].
+
+## Installing multiple integrations
+
+Installing more than one integration is a matter of adding the configuration information to your already existing `conf.yaml` file. Look up the required parameters for the new integration, and add it into your already existing `conf.yaml` file, and then follow the same steps to validate your configuration.
+
+## Security practices
+
+For information on how Datadog handles your data, and other security considerations, see our [Security documentation][21].
+
+## What's next?
+
+Now that you have your first integrations set up, you can start [exploring all of the metrics][22] being sent by Datadog to your application, and use these metrics to begin setting up [graphs][23] and [alerts][24] to monitor your data.
+
+Also check out our [Logs management][13], [APM][25], and [Synthetics][26] solutions.
+
+## Troubleshooting
```

Comment:
doesn't seem to be anything in this section

Reply:
Yeah, I was going to ask folks about this. What are the issues that people have that are common to any integration?

Reply:
Troubleshooting and integration validation process are quite similar.. Maybe we could just tell them to check if yaml file are valid, and then redirect to: https://docs.datadoghq.com/agent/troubleshooting/?tab=agentv6

Reply:
...the most common problem with integration setup is YAML file validation. If you YAML file is invalid an error message displays when checking the Agent's status...